### PR TITLE
Commands(OP): Make player non-case-sensitive

### PR DESCRIPTION
**Pull request by u9g**
/op u9g should still find U9G, now it does :)

### CODE REVIEW

File: src/lib/plugins/moderation.js [hunk 1]
```diff
@@ -98,10 +99,10 @@ module.exports.server = function (serv, settings) {
         if (!player.op) {
           player.op = true
 
-          player.chat(`§7§o[Server: Opped ${params[0]}]`)
-          return `Opped ${params[0]}`
+          player.chat(`§7§o[Server: Opped ${player.username}]`)
```
Comment (andriycraft) [inline src/lib/plugins/moderation.js:102]: Why [Server:.. ?
why not ${player.username || 'Server'

**Review by andriycraft — CHANGES_REQUESTED** [review]

File: src/lib/plugins/moderation.js (hunk 1 above)
Comment (u9g) [reply src/lib/plugins/moderation.js]: done in latest commit

**Review by andriycraft — APPROVED** [review]

**Review by andriycraft — CHANGES_REQUESTED** [review]
You forgot to make /deop command non case sensitive

**Comment by andriycraft** [comment]
@rom1504 what will happen if there are 2 players on the server: 'Kinash___' and 'KINASH___' then what player will be opped if i use /op Kotinash___

**Pull request merged**